docs/en/getting-started/example-datasets/reddit-comments.md

```diff
@@ -0,0 +1,636 @@
+
+```sql
+SELECT formatReadableQuantity(count())
+FROM reddit;
+```
+
+```response
+┌─formatReadableQuantity(count())─┐
+│ 85.97 million                   │
+└─────────────────────────────────┘
+```
+
+4. Let's see how many unique subreddits were in December of 2017:
+
+```sql
+SELECT uniqExact(subreddit)
+FROM reddit;
+```
+
+```response
+┌─uniqExact(subreddit)─┐
+│                91613 │
+└──────────────────────┘
+
+1 row in set. Elapsed: 1.572 sec. Processed 85.97 million rows, 367.43 MB (54.71 million rows/s., 233.80 MB/s.)
+```
+
+5. This query returns the top 10 subreddits (in terms of number of comments):
+
+```sql
+SELECT
+    subreddit,
+    count() AS c
+FROM reddit
+GROUP BY subreddit
+ORDER BY c DESC
+LIMIT 20;
+```
+
+```response
+┌─subreddit───────┬───────c─┐
+│ AskReddit       │ 5245881 │
+│ politics        │ 1753120 │
+│ nfl             │ 1220266 │
+│ nba             │  960388 │
+│ The_Donald      │  931857 │
+│ news            │  796617 │
+│ worldnews       │  765709 │
+│ CFB             │  710360 │
+│ gaming          │  602761 │
+│ movies          │  601966 │
+│ soccer          │  590628 │
+│ Bitcoin         │  583783 │
+│ pics            │  563408 │
+│ StarWars        │  562514 │
+│ funny           │  547563 │
+│ leagueoflegends │  517213 │
+│ teenagers       │  492020 │
+│ DestinyTheGame  │  477377 │
+│ todayilearned   │  472650 │
+│ videos          │  450581 │
+└─────────────────┴─────────┘
+
+20 rows in set. Elapsed: 0.368 sec. Processed 85.97 million rows, 367.43 MB (233.34 million rows/s., 997.25 MB/s.)
+```
+
+6. Here are the top 10 authors in December of 2017, in terms of number of comments posted:
+
+```sql
+SELECT
+    author,
+    count() AS c
+FROM reddit
+GROUP BY author
+ORDER BY c DESC
+LIMIT 10;
+```
+
+```response
+┌─author──────────┬───────c─┐
+│ [deleted]       │ 5913324 │
+│ AutoModerator   │  784886 │
+│ ImagesOfNetwork │   83241 │
+│ BitcoinAllBot   │   54484 │
+│ imguralbumbot   │   45822 │
+│ RPBot           │   29337 │
+│ WikiTextBot     │   25982 │
+│ Concise_AMA_Bot │   19974 │
+│ MTGCardFetcher  │   19103 │
+│ TotesMessenger  │   19057 │
+└─────────────────┴─────────┘
+
+10 rows in set. Elapsed: 8.143 sec. Processed 85.97 million rows, 711.05 MB (10.56 million rows/s., 87.32 MB/s.)
+```
+
+7.  We already inserted some data, but we will start over:
+
+```sql
+TRUNCATE TABLE reddit;
+```
+
+8. This is a fun dataset and it looks like we can find some great information, so let's go ahead and insert the entire dataset from 2005 to 2023. When you're ready, run this command to insert all the rows. (It takes a while - up to 17 hours!)
+
+```sql
+INSERT INTO reddit
+SELECT *
+FROM s3Cluster(
+    'default',
+    'https://clickhouse-public-datasets.s3.amazonaws.com/reddit/original/RC*',
+    'JSONEachRow'
+    )
+SETTINGS zstd_window_log_max = 31;
+```
+
+The response looks like:
+
+```response
+0 rows in set. Elapsed: 61187.839 sec. Processed 6.74 billion rows, 2.06 TB (110.17 thousand rows/s., 33.68 MB/s.)
```

Comment:
It is wrong, should be over 14 billion records.

Reply:
Resolved in https://github.com/ClickHouse/ClickHouse/pull/50419